aqua/QtPass/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               qt5 1.0
+PortGroup               qmake5 1.0
+
+github.setup            IJHack QtPass 1.4.0 v
+revision                0
+categories              aqua security
+license                 GPL-3+
+maintainers             nomaintainer
+platforms               darwin
```

Comment:
Sorry to return to the same issue, but if you prefer not to set specific platform restriction, then just drop it, `platforms darwin` is the default, it is redundant to explicitly write it.